docs/supported-community-modules.md

```diff
@@ -56,7 +56,7 @@ In addition to the ejected modules above, Microsoft has also added implementatio
 | <ins>[`rn-fetch-blob`](https://github.com/joltup/rn-fetch-blob)</ins> | [0.62](https://github.com/joltup/rn-fetch-blob/pull/701) | x |
 | <ins>[`react-native-code-push`](https://github.com/Microsoft/react-native-code-push)</ins> | 0.63 | x |
 | <ins>[`react-native-tts`](https://github.com/ak1394/react-native-tts)</ins> | 0.62 | x |
-| <ins>[`react-native-sketch-canvas`](https://github.com/creambyemute/react-native-sketch-canvas)</ins> | 0.63 | x |
+| <ins>[`react-native-sketch-canvas`](https://github.com/wwimmo/react-native-sketch-canvas)</ins> | 0.63 | x |
```

Comment:
is this change intentional?

Reply:
yes, CI was broken because it was trying to go to the old repo but that link redirects to this new one; the "link checker" failed/doesn't handle redirects so it went ❌

you can verify the behaviour by clicking here https://github.com/creambyemute/react-native-sketch-canvas

Reply:
gotcha thanks!